### ISSUE TITLE
Instant lost if color is changed

### ISSUE THREAD

**Issue by runxel** [Sh0wdown/UnofficialCrusaderPatch#234]
If you play a historical campaign the player will instantly lose the game if one changed the player color beforehand.
(Tested with the GOG version)

**Comment by Serpens66** [comment]
can not confirm this, I chose green and my troops in historical campaign are just green. But it can happen if you installed spectator mode I guess.

**Comment by runxel** [comment]
> But it can happen if you installed spectator mode I guess.
Ah yes. This could be the culprit as well.

**Comment by Sh0wdown** [comment]
> But it can happen if you installed spectator mode I guess.

Could you also confirm this, @runxel ?

**Comment by Serpens66** [comment]
its easier in german:
in einem Video von Udwin wo er den spectator modus vorstellt sieht man am Ende wie er eine Kreuzugsmission startet und instant verloren hat. Das war als 2.05 gerade neu war, weshalb es durchaus sein könnte, dass du das bereits gefixt hast. Aber es war jetzt das naheliegendste, da ich auch colours habe, aber keine spectator mode.

**Comment by Sh0wdown** [comment]
> in einem Video von Udwin wo er den spectator modus vorstellt sieht man am Ende wie er eine Kreuzugsmission startet und instant verloren hat.

Ist doch aber dann richtig so. Wenn nur noch die KI bzw. ein KI-Team über ist, hat sie halt gewonnen.

**Comment by Serpens66** [comment]
hab den zuschauermodus nicht getestet und weiß auch nicht, ob man den nach der installation noch ein/ausschalten kann.  Aber wenn er zwangsweise darin resultiert, dass man keine kreuzugs/kampagnen missionen mehr spielen kann und ihnen auch nicht zusehen kann, dann würde ich behaupten, dass das nicht richtig ist.  Entweder sollten diese Missionen vom Zuschauermodus ausgeschlossen sein, oder man sollte hier eben zu auch zuschauen können, wie die KI diese Mission versucht.

**Comment by runxel** [comment]
> 
> 
> > But it can happen if you installed spectator mode I guess.
> 
> Could you also confirm this, @runxel ?

Ja. Es liegt nicht am Farbenwechsel, sondern am Spectator Modus.
Habe den Titel daher angepasst.

**Comment by Heroesflorian** [comment]
@Serpens66 Ich finde es eigentlich nur logisch, da in den Missionen ja die KIs voreingestellt sind: Wenn Spieler 1 einfach nicht mitspielt, und keine KI voreingestellt ist und ja generell in den Missionen keine KI vom Spieler eingestellt/geändert werden kann... dann hat das Team von Spieler 1 eben instant verloren, sofern nicht noch Verbündete von Spieler 1 existieren. 
Beim Zuschauermodus wird ja der menschliche Spieler 1 entfernt, aber nicht automatisch durch ne KI ersetzt. Die KI rückt halt beim Hinzufügen in Custom-Matches grundsätzlich in den jeweils nächsten freien Spielerslot nach. Ohne menschlichen Spieler 1 ist der nächste freie Slot dann eben der für Spieler 1. 

Es wäre lediglich interessant, was passiert, wenn man Missionen startet, in denen Spieler 1 noch Verbündete hat - verliert man dann trotzdem instant oder nicht?
